resources/assets/less/skins/skin-black-dark.less

```diff
@@ -245,10 +245,6 @@ body {
 #details>div>div>div>table {
   background-color: transparent;
 }
-.dropdown-menu, .dropdown-menu>li>a {
```

Comment:
Do we not need a background color here? Or is that set somewhere else? (Forgive me, it's been a while since I dug into the less files for skins.)

Reply:
Would it be better to use the less variables?

Reply:
@Godmartinz can I get an update here?